Automattic/wp-calypso
# Update instructions to unlock domains in Squarespace

## Proposed Changes

Update Google Domains transfer instructions to Squarespace.

## Why are these changes being made?

All Google Domains that remained with Google were transferred to Squarespace. The Google Domains interface isn't accessible anymore, so we need to update our domain transfer instructions to use Squarespace.

## Testing Instructions

Visit `/setup/google-transfer/` and ensure the design changes added in p5VYbhFP6gBHwY1ACQK14d-fi match the content of this PR.

## Pre-merge Checklist
